bag/search/views.py

```diff
@@ -524,6 +527,9 @@ def _abstr_list(self, request: Request, q_select: AbstractSet[str]):
               paramType: query
         """
         query = request.query_params.get('q')
+        if 'features' in request.query_params:
+            features = request.query_params['features']
+            self.features = int(features) if features.isdigit() else 0
```

Comment:
Zou dit niet beter een foutmelding kunnen geven indien `not features.isdigit()`?

Reply:
And if an error message is more appropriate, arguably best to code in [EAFP](https://docs.python.org/3.8/glossary.html#term-eafp) style.

```python
try:
    self.features = int(features)
except ValueError as ve:
     # do whatever error reporting is appropriate
```